autoflake8/fix.py

```diff
@@ -499,24 +502,45 @@ def useless_pass_line_numbers(source: bytes) -> Iterator[int]:
             last_pass_row = start_row
             last_pass_indentation = get_indentation(line)
 
-        # Trailing "pass".
-        if (
-            is_pass
-            and previous_token_type != tokenize.INDENT
-            and not previous_line.rstrip().endswith(b"\\")
-        ):
-            yield start_row
+            is_trailing_pass = (
+                previous_token_type != tokenize.INDENT
+                and not previous_line.rstrip().endswith(b"\\")
+            )
+
+            is_pass_after_docstring = (
+                previous_token_type == tokenize.NEWLINE
+                and previous_line.rstrip().endswith(b'"""')
+            )
+
+            # Trailing "pass".
+            if is_trailing_pass:
+                if keep_pass_after_docstring and is_pass_after_docstring:
+                    pass
```

Comment:
Let's use `continue` to make the intent clear.
```suggestion
                    continue
```